gazelle/python/generate.go

```diff
@@ -476,6 +477,31 @@ func (py *Python) GenerateRules(args language.GenerateArgs) language.GenerateRes
 		result.Imports = append(result.Imports, pyTest.PrivateAttr(config.GazelleImportsKey))
 	}
 
+	if len(pyTestTargets) > 0 {
```

Comment:
This will make Gazelle generate a pytest main target and multiple py_test targets at the same time on the first run, and then generate another py_test target with the `main` pointing to the `__test__` target in the second run. People will have to run Gazelle twice to reach a stable state. At this state, every test file will be consumed by two `py_test` targets, so `bazel test //some:all` will execute all tests twice.